setup.py

```diff
@@ -37,6 +37,8 @@
             "jams",
             "requests",
             "pretty_midi >= 0.2.8",
+            "pyarrow",
+            "pandas"
```

Comment:
We had discussed in the past and opted for not including `pandas`. Also since `pyarrow` is used only for the `make_index` I would not add it as a dependency. See my comments below on moving to `csv`

For now, if @rabitt and @nkundiushuti agree with what I say above, please remove this dependencies from the `setup.py`

Reply:
@AngeloMendes if `parquet` is the only way of accessing the data we could still include it as `extras` (i.e. optional for users of the dataset)